### PR TITLE
Add fleet_default_member_config field to google_gke_hub_feature resource

### PR DESCRIPTION
**Pull request by sahsagar-google**
Fixes https://github.com/hashicorp/terraform-provider-google/issues/16222

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
gkehub: added `fleet_default_member_config` field to `google_gke_hub_feature` resource
```

### CODE REVIEW

File: mmv1/products/gkehub2/Feature.yaml [hunk 1]
```diff
@@ -170,6 +180,84 @@ properties:
...
+          - !ruby/object:Api::Type::Enum
+            name: management
+            description: 'Whether to automatically manage Service Mesh'
+            values:
+              - :MANAGEMENT_UNSPECIFIED
+              - :MANAGEMENT_AUTOMATIC
+              - :MANAGEMENT_MANUAL
+            required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: configmanagement
```
Comment (slevenick) [inline mmv1/products/gkehub2/Feature.yaml:200]: Can you specify more of these fields in the tests? We try to have as many of the fields set and then changed as possible.

**Comment by slevenick** [comment]
Looks like the mesh feature is not currently released

File: mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb [hunk 2]
```diff
@@ -347,6 +347,143 @@ resource "google_gke_hub_feature" "feature" {
...
+  service = "compute.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "test_container" {
+  service = "container.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_project_service" "test_gkehub" {
```
Comment (slevenick) [inline mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb:465]: For this test to work in VCR REPLAYING mode you will need to chain all of these google_project_service resources together via depends_on.

**Review by slevenick — CHANGES_REQUESTED** [review]
Please fix VCR by chaining resources

**Comment by sahsagar-google** [comment]
@slevenick fyi, I'm working on the comments and also adding more tests. Thank you

**Review by slevenick — CHANGES_REQUESTED** [review]
Test failing with: 
        Error: Error creating Feature: googleapi: Error 400: InvalidFieldError for field fleet_default_member_config: spec must be set


Are you able to run the test locally?

**Comment by sahsagar-google** [comment]
> Test failing with: Error: Error creating Feature: googleapi: Error 400: InvalidFieldError for field fleet_default_member_config: spec must be set
> 
> Are you able to run the test locally?

Investigating this. I think the visibility flag removal for the field is not rolled out fully. Let me deep-dive and reply back with the latest update. Thank you

**Review by slevenick — COMMENTED** [review]
Ah, looks like using gkeHubFeatureProjectSetupForGA in these tests messes with VCR, so I would re-add the skipIfVcr

**Review by slevenick — APPROVED** [review]

**Pull request merged**